src/task.js

```diff
@@ -92,3 +92,13 @@ clrBtn.addEventListener('click', () => {
   }
   localStorage.setItem('todoList', JSON.stringify(choresArr));
 });
+
+function check() {
+  document.getElementById('.todoCheckbox').checked = choresArr.complete(true);
+}
+
+function uncheck() {
+  document.getElementById('.todoChckbox').checked = choresArr.complete(false);
+}
+check();
+uncheck();
```

Comment:
- [ ] There is an error when I try to run your project it comes from here, please fix it 👌 
``` Cannot read properties of null (reading 'complete')```, which means 'choesArr' is 'null' at the beginning. to fix this I suggest adding a condition to check if choresArr is empty or not before calling the functions 👌

![image](https://user-images.githubusercontent.com/45231276/179381271-381c14f3-2fcc-4f60-a456-0eb019a26cb0.png)

- [ ] Please, make sure that there are no errors in the console 👌